doc/releases/changelog-dev.md

```diff
@@ -431,6 +431,10 @@
 
 <h3>Bug fixes 🐛</h3>
 
+* Plxpr-capture primitives now only return dynamically shaped arrays if their outputs
```

Comment:
Do we need to specify plxpr and capture? I'm not sure if there has been a shift in our convention for user-facing references to program capture
```suggestion
* Plxpr primitives now only return dynamically shaped arrays if their outputs
```